dc-cudami-editor/src/components/PublicationDatesForm.jsx

```diff
@@ -0,0 +1,75 @@
+import React, {Component} from 'react'
+import {Col, FormGroup, Label, Row} from 'reactstrap'
+import DatePicker from 'react-date-picker'
+import {useTranslation} from 'react-i18next'
+
+const PublicationDatesForm = (props) => {
+  const {t} = useTranslation()
+
+  const formatDate = (date) => {
+    if (date != undefined) {
```

Comment:
```suggestion
    if (date) {
```
dürfte es wohl auch tun, oder?

Reply:
Funktioniert damit auch :)